.php_cs

```diff
@@ -38,46 +38,47 @@ $config->setRules(
     [
         '@PSR2' => true,
         'align_multiline_comment' => ['comment_type' => 'all_multiline'],
+        'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'blank_line_before_statement' => [
             'statements' => ['return', 'continue', 'break', 'declare', 'throw', 'try'],
         ],
-        'cast_spaces' => ['space' => 'none'],
-        'combine_consecutive_issets' => true,
-        'combine_consecutive_unsets' => true,
+        'binary_operator_spaces' => true,
+        'cast_spaces' => ['space' => 'single'],
+        'class_attributes_separation' => ['elements' => ['method']],
         'compact_nullable_typehint' => true,
         'concat_space' => ['spacing' => 'one'],
-        'dir_constant' => true,
-        'function_to_constant' => true,
+        'declare_equal_normalize' => true,
+        'explicit_indirect_variable' => true,
+        'explicit_string_variable' => true,
+        'fully_qualified_strict_types' => true,
         'function_typehint_space' => true,
         'heredoc_to_nowdoc' => true,
         'include' => true,
         'linebreak_after_opening_tag' => true,
         'list_syntax' => ['syntax' => 'short'],
         'lowercase_cast' => true,
         'magic_constant_casing' => true,
-        'mb_str_functions' => true,
+        'method_chaining_indentation' => true,
         'method_separation' => true,
-        'modernize_types_casting' => true,
+        'multiline_whitespace_before_semicolons' => true,
         'native_function_casing' => true,
-        'native_function_invocation' => true,
         'new_with_braces' => true,
         'no_blank_lines_after_class_opening' => true,
         'no_blank_lines_after_phpdoc' => true,
         'no_empty_comment' => true,
         'no_empty_phpdoc' => true,
         'no_empty_statement' => true,
-        'no_extra_consecutive_blank_lines' => true,
-        'no_homoglyph_names' => true,
+        'no_extra_blank_lines' => true,
         'no_leading_import_slash' => true,
         'no_leading_namespace_whitespace' => true,
-        'no_mixed_echo_print' => ['use' => 'print'],
+        'no_mixed_echo_print' => true,
```

Comment:
👎

Reply:
Never saw anyone who would prefer `print` over `echo`. If you're favouring `print` because of its similarity to `print_r` or `printf` then it's wrong, because the former is a construct and the latter are functions.
Also, `echo` can stream multiple values, like so:

```php
echo 'Saying some words.', PHP_EOL, 'Hello world!';
```

Only advantage of `print` is ability to use it within expressions but I never saw anyone do it.

```php
<?php $condition ? print 'Yes' : null; ?>
```

Reply:
*shrug* It's personal preference here... I don't know why - but I can probably concede on this.